paddle/fluid/operators/controlflow/recurrent_op_helper.cc

```diff
@@ -241,12 +247,16 @@ void PrepareSafeEagerDeletionOnRecurrentOpAndRecurrentGradOp(
     const OpVariant *matched_fwd_op = nullptr;
     for (auto &fwd_op : recurrent_ops) {
       if (IsMatchedRecurrentOpAndRecurrentGradOp(fwd_op, bwd_op)) {
-        PADDLE_ENFORCE(matched_fwd_op == nullptr,
-                       "Found multiple matched recurrent op");
+        PADDLE_ENFORCE_EQ(matched_fwd_op, nullptr,
+                          platform::errors::PreconditionNotMet(
+                              "Found multiple recurrent forward op matches "
+                              "recurrent grad op"));
         matched_fwd_op = &fwd_op;
       }
     }
-    PADDLE_ENFORCE_NOT_NULL(matched_fwd_op, "Cannot find matched forward op");
+    PADDLE_ENFORCE_NOT_NULL(
+        matched_fwd_op,
+        platform::errors::PreconditionNotMet("Cannot find matched forward op"));
```

Comment:
It could be better to add `.` at last of sentence.

Reply:
Done